packages/contentful--cra-template-create-contentful-app/template/README.md

```diff
@@ -45,6 +45,33 @@ To make your app look and feel like Contentful use the following libraries:
 - [Forma 36](https://f36.contentful.com/) – Contentful's design system
 - [Contentful Field Editors](https://www.contentful.com/developers/docs/extensibility/field-editors/) – Contentful's field editor React components
 
+## Using the contentful-management SDK
+
+With the sdk and the contentful-management package installed,
+you can also create an instance of the contentful-management client using the `cmaAdapter`, which is part of the sdk, without additional configuration.
```

Comment:
"without additional configuration" is followed by some code with additional configuration. `{type: 'plain'...`

Reply:
changed it to without the access token, as this is what I meant with it